### DISCUSSION TITLE
sybase support

### DISCUSSION THREAD

**Discussion by pks888** [mithrandyr/SimplySql#134] [Ideas]

**Comment by mithrandyr** [comment]
@632575987  -- please provide more details.  Particularly helpful would be a link to a Nuget package that provides a .NET SDK for sybase.  I'm also not terribly familiar with Sybase -- [is this what you are referring to](https://www.sap.com/products/acquired-brands/what-is-sybase.html)?

**Comment by mithrandyr** [comment]
After reaching out to Sybase -- they have no ability/interest in supporting a non-commercial project like SimplySql in adding Sybase to its list of providers.  Unfortunately, without access to a developer version of Sybase I can't implement a test.  I'm closing this out, sorry.
